src/Neo.VM/EvaluationStack.cs

```diff
@@ -49,7 +49,7 @@ public IReadOnlyList<StackItem> this[Range range]
                 throw new ArgumentOutOfRangeException(nameof(range), "Range start must be less than or equal to end.");
 
             StackItem[] copyList = [.. _innerList];
-            List<StackItem> reverseList = [.. copyList.Reverse()];
+            List<StackItem> reverseList = [.. copyList.AsEnumerable().Reverse()];
```

Comment:
What's the difference?

Reply:
you get syntax error in IDE

Type 'void' can only be used in a spread element if it implements 'IEnumerable' or 'IEnumerable<T>', or if it has a suitable 'GetEnumerator' method whose return type has a 'Current' property and a 'MoveNext' method